specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/nova.json

```diff
@@ -0,0 +1,1054 @@
+          "items": {
+            "type": "object",
+            "properties": {
+              "datacenter": {
+                "type": "string",
+                "description": "The Cassandra data center this node resides in."
+              },
+              "status": {
+                "type": "string",
+                "description": "Indicates whether the node is functioning or not.",
+                "enum": [
+                  "Up",
+                  "Down"
+                ],
+                "x-ms-enum": {
+                  "name": "NodeStatus"
+                }
+              },
+              "state": {
+                "type": "string",
+                "description": "The state of the node in relation to the cluster.",
+                "enum": [
+                  "Normal",
+                  "Leaving",
+                  "Joining",
+                  "Moving",
+                  "Stopped"
+                ],
+                "x-ms-enum": {
+                  "name": "NodeState"
+                }
+              },
+              "address": {
+                "type": "string",
+                "description": "The node's URL."
+              },
+              "load": {
+                "type": "string",
+                "description": "The amount of file system data in the data directory (e.g., 47.66 KB), excluding all content in the snapshots subdirectories. Because all SSTable data files are included, any data that is not cleaned up (such as TTL-expired cell or tombstoned data) is counted."
+              },
+              "tokens": {
+                "type": "integer",
+                "format": "int32",
+                "description": "The number of tokens set for the node."
+              },
+              "owns": {
+                "type": "number",
+                "format": "double",
+                "description": "The percentage of the data owned by the node per datacenter times the replication factor (e.g., 33.3, or null if the data is not available). For example, a node can own 33% of the ring, but shows 100% if the replication factor is 3. For non-system keyspaces, the endpoint percentage ownership information is shown."
+              },
+              "hostId": {
+                "type": "string",
+                "description": "The network ID of the node."
+              },
+              "rack": {
+                "type": "string",
+                "description": "The rack this node is part of."
+              }
+            }
+          }
+        }
+      }
+    },
+    "BackupResource": {
+      "description": "A restorable backup of a Cassandra cluster.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "properties": {
+            "timestamp": {
```

Comment:
May want to consider adding systemData to this resource (already defined in common types). Details here: https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html?q=SystemData

Reply:
CosmosDB hasn't added it in general, so we're planning to add it at the same time that the rest of the resource provider gets it.